sql/src/main/java/io/crate/operation/collect/sources/InformationSchemaIterables.java

```diff
@@ -136,14 +101,9 @@ public boolean apply(@Nullable RoutineInfo input) {
         ColumnsIterator(TableInfo ti) {
             context.tableInfo = ti;
             context.ordinal = 0;
-            columns = FluentIterable.from(ti).filter(new Predicate<Reference>() {
-                @Override
-                public boolean apply(@Nullable Reference input) {
-                    return input != null
-                           && !input.ident().columnIdent().isSystemColumn()
-                           && input.valueType() != DataTypes.NOT_SUPPORTED;
-                }
-            }).iterator();
+            columns = StreamSupport.stream(ti.spliterator(), false)
+                .filter(i -> !i.ident().columnIdent().isSystemColumn() && i.valueType() != DataTypes.NOT_SUPPORTED)
```

Comment:
tables don't have `null` columns, the null check was there because the Predicate interface has a Nullable annotation. I can re-add it so that static analyzers don't complain